### PR TITLE
[CI Fix] Fix flake8 errors.

### PR DESCRIPTION
**Pull request by 0mdc**
## Motivation and Context

This fixes two flake8 errors that surfaced after a dependency upgrade.

## How Has This Been Tested

Tested on CI.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Skylion007** [comment]
Sorry, that my commit to flake8-comprehensions. Meant to go and ruff on habitat-lab as well.

File: tests/test_semantic_scene.py [hunk 1]
```diff
@@ -38,7 +38,6 @@ def test_semantic_scene(scene, make_cfg_settings):
     if not osp.exists(scene):
         pytest.skip("Skipping {}".format(scene))
 
-    make_cfg_settings = {k: v for k, v in make_cfg_settings.items()}
```
Comment (Skylion007) [inline tests/test_semantic_scene.py:41]: Is this suppose to be a `copy()`?

Comment (0mdc) [reply tests/test_semantic_scene.py:41]: The `make_cfg_settings` passed to the test is already a copy.

https://github.com/facebookresearch/habitat-sim/blob/fed4be9bcae818dc5e76aaa548f33a43c5ff6734/tests/conftest.py#L25

```
def make_cfg_settings():
    import habitat_sim.utils.settings

    cfg = habitat_sim.utils.settings.default_sim_settings.copy()
    {...}
    return cfg
```

Comment (0mdc) [reply tests/test_semantic_scene.py:41]: The `make_cfg_settings` passed to the test is already a copy.

https://github.com/facebookresearch/habitat-sim/blob/fed4be9bcae818dc5e76aaa548f33a43c5ff6734/tests/conftest.py#L25

```
def make_cfg_settings():
    import habitat_sim.utils.settings

    cfg = habitat_sim.utils.settings.default_sim_settings.copy()
    {...}
    return cfg
```

**Review by Skylion007 — APPROVED** [review]

**Pull request merged**